backend/src/routes/v0/server_validation.py

```
@@ -0,0 +1,116 @@
+"""
+Server configuration validation logic.
+
+This module contains all validation functions for different server types
+and their configurations, providing clear error messages and validation rules.
+"""
+
+from typing import List, Dict, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)